pkg/lib/infra/sms/custom.go

```diff
@@ -30,19 +30,38 @@ func NewCustomClient(c *config.CustomSMSProviderConfig, d SMSDenoHook, w SMSWebH
 }
 
 type SendSMSPayload struct {
-	To   string `json:"to"`
-	Body string `json:"body"`
+	To                string             `json:"to"`
+	Body              string             `json:"body"`
+	AppID             string             `json:"app_id"`
+	TemplateName      string             `json:"template_name"`
```

Comment:
MessageType string `json:"message_type"`
TemplateName string `json:"template_name"` (basename of the logic path)